src/main/java/org/folio/circulation/domain/LoanRepresentation.java

```diff
@@ -39,11 +39,11 @@ public JsonObject extendedLoan(Loan loan) {
       extendedRepresentation.remove(LoanProperties.BORROWER);
     }
 
-    if (loan.getLoanPolicy() != null) {
-      additionalLoanPolicyProperties(extendedRepresentation, loan.getLoanPolicy());
-    } else {
-      extendedRepresentation.remove(LoanProperties.LOAN_POLICY);
-    }
+    addPolicy(extendedRepresentation, loan.getLoanPolicy(), LoanProperties.LOAN_POLICY);
+
+    addPolicy(extendedRepresentation, loan.getOverdueFinePolicy(), LoanProperties.OVERDUE_FINE_POLICY);
+
+    addPolicy(extendedRepresentation, loan.getLostItemPolicy(), LoanProperties.LOST_ITEM_POLICY);
```

Comment:
Personally, I would use a static import for the `LoanProperties` values to reduce the line length and improve readability

Reply:
Changed